build.js

```diff
@@ -120,6 +115,11 @@ fs.writeFileSync(
 `
 );
 
+if (currentPlatform() == "macOS" && drived_h_changed) {
+  // Make sure the main executable gets rebuilt with the new build ID.
```

Comment:
shouldn't normal ninja deps ensure this already?

Reply:
I'm not sure, I don't remember adding this part 😬